docs/docs-content/release-notes/release-notes.md

```diff
@@ -108,21 +115,21 @@ to learn more about this release's new features and improvements.
 
```

Comment:
To the line above: there are two periods after `untampered` :arrow_up:

Reply:
Nice catch